pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.black]
+line-length = 120
+target-version = ['py38']
+include = '\.pyi?$'
+exclude = '''
```

Comment:
Nitpick: It looks like `black` [automatically ignores files in the gitignore](https://github.com/psf/black/pull/878) so you might be able to clean this up a bit.